.travis.yml

```diff
@@ -21,4 +21,8 @@ script:
   - cabal build
   - cabal test
   - cabal install
+  - ([ "$GHCVER" = "7.6.3" ] || cabal configure --enable-tests --enable-executable-dynamic)
+  - ([ "$GHCVER" = "7.6.3" ] || cabal build)
+  - ([ "$GHCVER" = "7.6.3" ] || cabal test)
+  - ([ "$GHCVER" = "7.6.3" ] || cabal install)
```

Comment:
it's testing the dynamic executable as well... but looking at the build times this is probably not a good way of doing it... it could be part of the build matrix instead.